### PR TITLE
feat: add generate API and Helm doc process with zh content update

### PR DESCRIPTION
**Pull request by wilsonwu**
**What type of PR is this?**
The API docs don't need to translate, so add generate API process with zh content update:
1. add a copy cmd to zh content after API generation both for api and helm doc.
2. update the `docs-api-headings` support zh content update

Not sure it is OK or not, please help to review.

### CODE REVIEW

File: tools/make/docs.mk [hunk 1]
```diff
@@ -40,6 +40,7 @@ helm-readme-gen: $(tools/helm-docs)
 	@ImageRepository=docker.io/envoyproxy/gateway ImageTag=latest ImagePullPolicy=IfNotPresent envsubst < charts/gateway-helm/values.tmpl.yaml > ./charts/gateway-helm/values.yaml # use production ENV to generate helm api doc
 	$(tools/helm-docs) charts/gateway-helm/ -f values.yaml -o api.md
 	mv charts/gateway-helm/api.md site/content/en/latest/install/api.md
+	cp site/content/en/latest/install/api.md site/content/zh/latest/install/api.md
```
Comment (shawnh2) [inline tools/make/docs.mk:43]: can you add a comment about why we are doing this?

**Review by shawnh2 — APPROVED** [review]
LGTM! Thanks

File: tools/make/docs.mk (hunk 1 above)
Comment (wilsonwu) [reply tools/make/docs.mk:43]: Done

File: tools/make/docs.mk [hunk 2]
```diff
@@ -41,6 +41,8 @@ helm-readme-gen: $(tools/helm-docs)
 	$(tools/helm-docs) --template-files=tools/helm-docs/readme.gotmpl charts/gateway-helm/ -f values.yaml -o README.md
 	$(tools/helm-docs) --template-files=tools/helm-docs/api.gotmpl charts/gateway-helm/ -f values.yaml -o api.md
 	mv charts/gateway-helm/api.md site/content/en/latest/install/gateway-helm-api.md
+  # below line copy command for sync English api doc into Chinese
```
Comment (zirain) [inline tools/make/docs.mk:44]: miss `\t` at the begin?

Comment (wilsonwu) [reply tools/make/docs.mk:44]: Done

**Comment by wilsonwu** [comment]
@zirain all up to date, please help to review.

**Review by zirain — APPROVED** [review]

**Review by zhaohuabing — APPROVED** [review]
LGTM thanks!

**Pull request merged**